test/VectorInstructions/insert_element_symbolic.c

```diff
@@ -0,0 +1,22 @@
+// REQUIRES: geq-llvm-3.4
```

Comment:
Again, I don't think testing unsupported features is important, and we also need to keep our regression tests from growing too much.